roles/openshift_logging/tasks/generate_certs.yaml

```diff
@@ -125,7 +125,7 @@
     - system.logging.mux
   loop_control:
     loop_var: node_name
-  when: openshift_logging_use_mux
+  when: openshift_logging_use_mux | bool
```

Comment:
I think this broke the install. I think generate_jks.yaml expects this to exist no matter what.